src/main/kotlin/no/skatteetaten/aurora/boober/service/CantusService.kt

```diff
@@ -94,8 +122,22 @@ class CantusService(
             "${cmd.fromRegistry}/${cmd.name}:${cmd.from}",
             "${cmd.toRegistry}/${cmd.name}:${cmd.to}"
         )
-        val resultEntity = client.post(body = cantusCmd, type = JsonNode::class, url = "/tag")
-        logger.info("Response from cantus code=${resultEntity.statusCode} body=${resultEntity.body}")
-        return TagResult(cmd, resultEntity.body, resultEntity.statusCode.is2xxSuccessful)
+        val response = runCatching {
+            client.post(body = cantusCmd, type = JsonNode::class, url = "/tag")
+        }.also(::logResult)
+            .getOrNull()
+
+        val success = response != null
+
+        return TagResult(cmd, response?.body, success)
     }
 }
+
+// TODO: should we log as error instead?
```

Comment:
IMHO no, logging errors should be used when you need to do something about it in this service, and that is not the case here